results/cmd/api/main.go

```diff
@@ -43,29 +46,36 @@ func main() {
 		log.Fatalf("failed to open the results.db: %v", err)
 	}
 	defer db.Close()
-	srv := &server{db: db}
-
+	// Create cel enviroment for filter
+	env, err := cel.NewEnv(
+		cel.Types(&pb.TaskRun{}),
+		cel.Declarations(decls.NewIdent("taskrun", decls.NewObjectType("tekton.TaskRun"), nil)),
+	)
+	if err != nil {
+		log.Fatalf("failed to create environment for filter: %v", err)
+	}
+	srv := &server{db: db, env: env}
 	// Listen for gRPC requests.
 	lis, err := net.Listen("tcp", port)
 	if err != nil {
 		log.Fatalf("failed to listen: %v", err)
 	}
 	s := grpc.NewServer()
 	pb.RegisterResultsServer(s, srv)
+	reflection.Register(s)
```

Comment:
question: why do we need this?

Reply:
Without this line, I cannot use grpc-cli:)

Reply:
I will add a comment for this

Reply:
done